digitalocean/datasource_digitalocean_kubernetes_cluster_test.go

```diff
@@ -16,9 +16,9 @@ func TestAccDataSourceDigitalOceanKubernetesCluster_Basic(t *testing.T) {
 	var k8s godo.KubernetesCluster
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckDigitalOceanKubernetesClusterDestroy,
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
```

Comment:
```suggestion
		ProviderFactories: testAccProviderFactories,
		ExternalProviders: map[string]resource.ExternalProvider{
			"kubernetes": {
				Source:            "hashicorp/kubernetes",
				VersionConstraint: "1.13.2",
			},
		},
```

This test was failing due to the removal of the Kubernetes provider. The upgrade guide seems to imply that the provider just needs to be declared in the test config, but that doesn't seem to be the case.

https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html#a-new-acceptance-testing-driver

Looks like it needs to be defined with:

https://github.com/hashicorp/terraform-plugin-sdk/blob/v2.0.3/helper/resource/testing.go#L304-L310